packages/eas-build-job/src/ios.ts

```diff
@@ -80,6 +81,7 @@ const BuilderEnvironmentSchema = Joi.object({
 export interface Job {
   type: Workflow;
   projectArchive: ArchiveSource;
+  buildResourceClass: BuildResourceClass;
```

Comment:
value in schema is optional and in type it's required, this is not correct

Reply:
fixed for both iOS and Android